src/gccrs/rustc_options.rs

```diff
@@ -30,6 +30,13 @@ impl RustcOptions {
             "KIND[=PATH]",
         );
         options.optmulti("", "crate-type", "Type of binary to output", "TYPE");
+        options.optmulti(
```

Comment:
You could use `#![feature(rustc_private)]` and reuse the rustc config parser. But then you'd would have to use a nightly toolchain to compile this crate...

Reply:
See https://github.com/Rust-GCC/cargo-gccrs/issues/46.